h/accounts/schemas.py

```diff
@@ -142,7 +142,7 @@ class RegisterSchema(CSRFSchema):
             validators.Length(min=USERNAME_MIN_LENGTH, max=USERNAME_MAX_LENGTH),
             colander.Regex(
                 USERNAME_PATTERN,
-                msg=_("Must have only letters, numbers, periods, and " "underscores."),
```

Comment:
I don't know if this was like this for a reason. I can't think of one.

Reply:
This is a result of black backing up what used to be on two lines I think.

As a bonus it makes it impossible to grep for!